src/middleware/serverMiddlewareLoader.middleware.ts

```diff
@@ -27,6 +32,20 @@ export default class ServerMiddlewareLoader {
     app.use(bodyParser.urlencoded({ extended: true }))
     app.use(cookieParser())
     app.use(createLoggerMiddleware(APP_NAME))
+    app.use(helmet.hidePoweredBy())
+    app.use(
```

Comment:
I'd replace this with `this.setHeaders(app)` to keep the helmet stuff into it's own dedicated private method